site/ko/blog/new-in-devtools-95/index.md

```diff
@@ -102,55 +100,60 @@ link.search = `?access=${link.access}`;
 Chromium issues: [1076820](https://crbug.com/1076820), [1119900](https://crbug.com/1119900)
 
 
-<!-- ## Lighthouse 8.4 in the Lighthouse panel {: #lighthouse } -->
+## Lighthouse 8.4 {: #lighthouse }
 
-<!-- The **Lighthouse** panel is now running Lighthouse 8.4. Lighthouse will now detect if the [Largest Containful Paint (LCP)](https://web.dev/lcp) element was a lazy-loaded image and recommend removing the `loading` attribute from it. -->
+Lighthouse 패널이 이제 Lighthouse 8.4로 업데이트 되었습니다. 
+이제 Lighthouse에서 [Largest Containful Paint (LCP)](https://web.dev/lcp) 요소가 레이지 로딩된 이미지인 지 감지하고
+해당 요소에서 `loading` 속성을 제거할 것을 권장합니다.
 
-<!-- Check out the [What’s new in Lighthouse 8.4](/blog/lighthouse-8-4/) for more details on the updates. -->
+전체 변경사항은 [What’s new in Lighthouse 8.4](/blog/lighthouse-8-4/) (영문) 를 참고하세요.
 
-{% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/u9nepJj3wgpMgoNxSaDZ.png", alt="The lazy-loaded LCP audit in a Lighthouse report", width="800", height="502", class="screenshot" %}
+{% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/u9nepJj3wgpMgoNxSaDZ.png", alt="Lighthouse 보고서에서 레이지 로딩된 LCP 측정", width="800", height="502", class="screenshot" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/649a979e4de2cf38430e46e7198b11ba8a830388 #}
 
 Chromium issue: [772558](https://crbug.com/772558)
 
 
-<!-- ## Sort snippets in the Sources panel {: #snippets } -->
+## 소스 패널에서 스니펫 정렬 {: #snippets }
 
-<!-- The [snippets](/docs/devtools/javascript/snippets/) in the **Snippets** pane under the **Sources** panel are now sorted alphabetically. Previously, it’s not sorted. -->
+**소스** 패널에 존재하는 **스니펫** 영역에서 [스니펫](/docs/devtools/javascript/snippets/) 을 알파벳순으로 정렬됩니다.
```

Comment:
Suggestion: **소스** 패널에 존재하는 **스니펫** 영역의 [스니펫](/docs/devtools/javascript/snippets/) 은 현재 알파벳 순으로 정렬됩니다.

Reply:
'현재' sounds like it used to be like that. How about '이제' to make it feel more like it WILL be as contrast to how previously it WAS. And '존재하는' sounds a bit too serious. :) 

Suggestion:
소스 패널 안에 있는 스니펫 영역의 스니펫들은 이제 알파벳 순으로 정렬됩니다. (생략: 이전에는 정렬되지 않았었습니다.)